wcivf/apps/elections/views/postcode_view.py

```diff
@@ -82,6 +83,21 @@ def get_todays_ballots(self):
             ballot for ballot in self.ballots if ballot.election.is_election_day
         ]
 
+    def get_referendums(self):
+        """
+        Yield all referendums associated with the ballots for this postcode.
+        After 6th May return an empty list to avoid displaying unwanted
```

Comment:
This makes me think of two things:

1. We should show these for some weeks after election day - people come back after to see what happened.
2. We should consider telling people what happened!

Reply:
Good points - I feel like there is a few different things to think about with this, from what we display to how we display it, and more basic things like the wording in the card. So I think it probably merits a separate issue/PR? 

So for now if you are happy I thought I'd leave in the hard 'stop displaying after 6th May' so that this can be merged? And I'll add a new issue for further changes.

Reply:
Yep, that words for me! Are you ok to make those issues?